mslib/mscolab/conf.py

```diff
@@ -79,7 +79,7 @@ class mscolab_settings(object):
         enable_basic_http_authentication = False
 
         # enable verification by Mail
-        USER_VERIFICATION = False
+        MAIL_ENABLED = False
```

Comment:
I have a valid email configuration and it sends me the confirmation email also when MAIL_ENABLED is False.

I think it should silently create the account without mailing me the verification token.

Reply:
using that link gives a 500

```
ERROR: Exception on /confirm/TOKEN [GET]
Traceback (most recent call last):
  File "/home/icg105/mambaforge/envs/mssenv/lib/python3.10/site-packages/flask/app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/icg105/mambaforge/envs/mssenv/lib/python3.10/site-packages/flask/app.py", line 1823, in full_dispatch_request
    return self.finalize_request(rv)
  File "/home/icg105/mambaforge/envs/mssenv/lib/python3.10/site-packages/flask/app.py", line 1842, in finalize_request
    response = self.make_response(rv)
  File "/home/icg105/mambaforge/envs/mssenv/lib/python3.10/site-packages/flask/app.py", line 2134, in make_response
    raise TypeError(
TypeError: The view function for 'confirm_email' did not return a valid response. The function either returned None or ended without a return statement.
```

Reply:
the url for Recover Your Password is protected by the 403 status page. :)  :heart: